test/Listener/LocatorRegistrationListenerTest.php

```diff
@@ -13,7 +13,7 @@
 use Zend\EventManager\EventManager;
 use Zend\EventManager\SharedEventManager;
 use Zend\ModuleManager\Listener\LocatorRegistrationListener;
-use Zend\ModuleManager\Listener\ModuleResolverListener;
+use Zend\ModuleManager\Listener\DefaultListenerAggregate;
```

Comment:
Alphabetical order please.

Reply:
ugh, this is weird.  I don't remember doing that change